client/src/views/SelfReview/SelfReview.js

```diff
@@ -1,6 +1,263 @@
+    setReviewDetails(details)
+    setProjectDetails([projectsArr.join(', ')])
+    setShowDetailsDialog(true)
+  }
+  const selfReviewArray = []
+  let filteredEmployee
+  if (selfReviewData) {
+    filteredEmployee = selfReviewData.filter(
+      cls =>
+        (cls.employee.firstname +
+          ' ' +
+          cls.employee.lastname)
+          .toLowerCase()
+          .includes(selectedEmployee.toLowerCase().trim()) &&
+        cls.status !== 'Inactive'
+    )
+    filteredEmployee.map((review, key) => {
+      let projectsArr = review.projects.map(item => item.title)
+      selfReviewArray.push([
+        <span
+          className={classes.showPointer}
+          onClick={() => viewDetailHandler(review, projectsArr)}
+        >
+          {review.employee.firstname +
+            ' ' +
+            review.employee.lastname}
+        </span>,
+        projectsArr.join(', '),
+        formatDate(review.from_date),
+        formatDate(review.to_date),
+        formatDate(review.due_to),
+        review.status
+      ])
+      return 1
+    })
+  }
+
+  const changeHandler = e => {
+    setselectedEmployee(e.target.value)
+  }
+  const createPeerHandler = () => {
+    setSelfReviewInfo('')
+    setIsRedirectForm(true)
+  }
+  const detailsSwitchHandler = () => {
+    setIsRedirectForm(false)
+  }
+  const updateUser = (val, k) => {
+    setSelfReviewInfo(filteredEmployee[k])
+    setIsRedirectForm(true)
+  }
+
+  const deleteUser = (val, k) => {
+    setDeleteId(filteredEmployee[k]._id)
+    setShowDelDialog(true)
+  }
+  const handleYesDelete = () => {
+    dispatch(deleteSelfReview(deleteId))
+    setShowDelDialog(false)
+  }
+  return (
+    <div>
+      {isRedirectForm ? (
+        <SelfReviewForm
+          selfReviewInfo={selfReviewInfo}
+          clickHandler={detailsSwitchHandler}
+        ></SelfReviewForm>
+      ) : (
+          <GridContainer>
+            <Grid xs={1} sm={1} md={1} className={classes.grid} item>
+              <InputLabel>Search By:</InputLabel>
+            </Grid>
+            <GridItem xs={5} sm={5} md={5}>
+              <FormControl className={classes.formControl}>
+                <InputLabel htmlFor="SelectEmployee"> Select Employee</InputLabel>
+                <Select
+                  value={selectedEmployee}
+                  onChange={changeHandler}
+                  inputProps={{
+                    name: 'SelectEmployee',
+                    id: 'SelectEmployee'
+                  }}
+                >
+                  <MenuItem className={classes.hoverEffect} value="">
+                    <em>None</em>
+                  </MenuItem>
+                  {employeeData
+                    ? employeeData.map((prop, key) => {
+                      return prop.status !== 'Inactive' ? (
+                        <MenuItem
+                          className={classes.hoverEffect}
+                          value={prop.firstname + ' ' + prop.lastname}
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
did in PR#31